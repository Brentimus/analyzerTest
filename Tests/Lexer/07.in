AnD 
ArrAY 
as 
ASM 
BegiN 
BREAK 
CaSe 
CLASS 
CONST 
CONSTRUCTOR 
CONTINUE 
DESTRUCTOR 
DEC 
DIV 
DISPOSE 
DO 
DOWNTO 
ELSE 
END 
EXCEPT 
EXIT 
EXPORTS 
FALSE 
FILE 
FINALIZATION 
FINALLY 
FLOAT 
FOR 
FUNCTION 
GOTO 
IF 
IMPLEMENTATION 
IN 
INC 
INLINE 
INHERITED 
INITIALIZATION 
INTERFACE 
IS 
LABEL 
LIBRARY 
MOD 
NEW 
NIL 
NOT 
OBJECT 
OF 
ON 
OPERATOR 
OR 
OUT 
PACKED 
PROCEDURE 
PROGRAM 
PROPERTY 
REAL 
RECORD 
REPEAT 
REAISE 
SELF 
SET 
SHL 
SHR 
STRING 
THEN 
THREADVAR 
TO 
TRUE 
TRY 
TYPE 
UNIT 
UNTIL 
USES 
VAR 
WHILE 
WITH 
XOR