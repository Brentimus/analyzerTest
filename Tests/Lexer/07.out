1  1  Keyword  AND  AnD
2  1  Keyword  ARRAY  ArrAY
3  1  Keyword  AS  as
4  1  Keyword  ASM  ASM
5  1  Keyword  BEGIN  BegiN
6  1  Keyword  BREAK  BREAK
7  1  Keyword  CASE  CaSe
8  1  Keyword  CLASS  CLASS
9  1  Keyword  CONST  CONST
10  1  Keyword  CONSTRUCTOR  CONSTRUCTOR
11  1  Keyword  CONTINUE  CONTINUE
12  1  Keyword  DESTRUCTOR  DESTRUCTOR
13  1  Keyword  DEC  DEC
14  1  Keyword  DIV  DIV
15  1  Keyword  DISPOSE  DISPOSE
16  1  Keyword  DO  DO
17  1  Keyword  DOWNTO  DOWNTO
18  1  Keyword  ELSE  ELSE
19  1  Keyword  END  END
20  1  Keyword  EXCEPT  EXCEPT
21  1  Keyword  EXIT  EXIT
22  1  Keyword  EXPORTS  EXPORTS
23  1  Keyword  FALSE  FALSE
24  1  Keyword  FILE  FILE
25  1  Keyword  FINALIZATION  FINALIZATION
26  1  Keyword  FINALLY  FINALLY
27  1  Keyword  FLOAT  FLOAT
28  1  Keyword  FOR  FOR
29  1  Keyword  FUNCTION  FUNCTION
30  1  Keyword  GOTO  GOTO
31  1  Keyword  IF  IF
32  1  Keyword  IMPLEMENTATION  IMPLEMENTATION
33  1  Keyword  IN  IN
34  1  Keyword  INC  INC
35  1  Keyword  INLINE  INLINE
36  1  Keyword  INHERITED  INHERITED
37  1  Keyword  INITIALIZATION  INITIALIZATION
38  1  Keyword  INTERFACE  INTERFACE
39  1  Keyword  IS  IS
40  1  Keyword  LABEL  LABEL
41  1  Keyword  LIBRARY  LIBRARY
42  1  Keyword  MOD  MOD
43  1  Keyword  NEW  NEW
44  1  Keyword  NIL  NIL
45  1  Keyword  NOT  NOT
46  1  Keyword  OBJECT  OBJECT
47  1  Keyword  OF  OF
48  1  Keyword  ON  ON
49  1  Keyword  OPERATOR  OPERATOR
50  1  Keyword  OR  OR
51  1  Keyword  OUT  OUT
52  1  Keyword  PACKED  PACKED
53  1  Keyword  PROCEDURE  PROCEDURE
54  1  Keyword  PROGRAM  PROGRAM
55  1  Keyword  PROPERTY  PROPERTY
56  1  Keyword  REAL  REAL
57  1  Keyword  RECORD  RECORD
58  1  Keyword  REPEAT  REPEAT
59  1  Keyword  REAISE  REAISE
60  1  Keyword  SELF  SELF
61  1  Keyword  SET  SET
62  1  Keyword  SHL  SHL
63  1  Keyword  SHR  SHR
64  1  Keyword  STRING  STRING
65  1  Keyword  THEN  THEN
66  1  Keyword  THREADVAR  THREADVAR
67  1  Keyword  TO  TO
68  1  Keyword  TRUE  TRUE
69  1  Keyword  TRY  TRY
70  1  Keyword  TYPE  TYPE
71  1  Keyword  UNIT  UNIT
72  1  Keyword  UNTIL  UNTIL
73  1  Keyword  USES  USES
74  1  Keyword  VAR  VAR
75  1  Keyword  WHILE  WHILE
76  1  Keyword  WITH  WITH
77  1  Keyword  XOR  XOR
77  4  Eof